cmd/kubeadm/app/cmd/init_test.go

```diff
@@ -157,3 +185,14 @@ func TestNewInitData(t *testing.T) {
 		})
 	}
 }
+
+func expectedInitIgnorePreflightErrors(expected sets.String) func(t *testing.T, data *initData) {
```

Comment:
You can have the `expected` parameter as a string slice and then convert it to a set with `NewString`. This will make the calls less verbose by omitting the extra `NewString` call in them.

Reply:
Indeed better, well spotted 👍 
Will push soonish.